packages/pds/src/services/moderation/views.ts

```diff
@@ -94,7 +94,7 @@ export class ModerationViews {
       return {
         did: r.did,
         handle: r.handle,
-        account: email ? { email } : undefined,
+        account: email ?? undefined,
```

Comment:
I think we want to name the property `email`:
```suggestion
        email: email ?? undefined,
```

Reply:
oh yup lol good call